store/store.go

```diff
@@ -142,6 +163,16 @@ func NewStore(
 			cacheData[parts[0]] = parts[1]
 		}
 	}
+	st.cacheData = cacheData
```

Comment:
Should writes to cacheData be synchronized with accesses with a mutex or something?

(Sorry for the delayed response.)

Reply:
Use atomic.StorePointer to assign the new cache into store